util/ctxutil.go

```diff
@@ -0,0 +1,18 @@
+package util
+
+import (
+	"context"
+	"time"
+)
+
+// NewClonedContext returns a context with the same Values
```

Comment:
Allows to clone a context values, without inheriting its cancellation setup.
Handly for spinning goroutines out of the request scope.

Reply:
Clever!